### ISSUE TITLE
Wechseln der Models

### ISSUE THREAD

**Issue by redscarwolf** [redscarwolf/VR-Robot#3]
# Mögliche Impl:
## per tastendruck
boooooring

## Watson SpeechToText Service
Der Service ist sowieso schon eingebaut. Man müsste also den erkannten Text nur überprüfen:
`if (recognizedText == "change model") { ... }`. Noch zu klären ist, wie das Model dann tatsächlich ausgetauscht wird, so dass der gleiche Animator nun auf ein anderes Objekt zeigt.

## Watson Conversation Service
Oder die Deluxe-Version, falls dafür Zeit bleibt. Den bereits erkannten Text durch den Watson Conversation Service schleifen, so dass jedes Model explizit ausgewählt werden kann und mit mehreren Synonymen bezeichnet. 

Also ein Intent ("change model", "switch robot", "change to", ...) 

und mehrere Entities, je eine pro Model ("Lisa's robot", "Lisa's model", "Lisa's bot", "Slug-Bot", "Slug-Bot 5000", "Lisa Robot", ...)

(Alles, was nicht vom Conversation Service erkannt wird, weil Confidence zu niedrig, wird dann wie gehabt zur Interaktion mit dem Bot genutzt?)

**Comment by nielsboecker** [comment]
Außerdem noch TODO: 

- Dropbox o. Ä. anlegen, KDler instruieren
- Zusammenspiel der beiden Skripte überprüfen. Evtl. `Bored.cs` umbenennen zu etwas generischerem.
- mit Vive testen

**Comment by DanielHipp** [comment]
Jo, also ich hab den switch jetzt mit button eingefügt. Wenn einer von euch lust hat kann er gerne noch das if statement einfügen und den inhalt des scriptes verschieben. So wies jetzt ist wird auf leertaste der bot gewechselt
